supertokens_python/utils.py

```diff
@@ -175,3 +178,18 @@ def execute_async(mode: str, func: Callable[[], Coroutine[Any, Any, None]]):
 
 def frontend_has_interceptor(request: BaseRequest) -> bool:
     return get_rid_from_request(request) is not None
+
+
+def deprecated_warn(msg: str):
+    warnings.warn(msg, DeprecationWarning, stacklevel=2)
+
+
+def handle_httpx_client_exceptions(e: Exception, input_: Dict[str, Any]):
+    if isinstance(e, HTTPStatusError):
+        res: Response = e.response  # type: ignore
+        log_debug_message("Error status: %s", res.status_code)  # type: ignore
+        log_debug_message("Error response: %s", res.json())
+    else:
+        log_debug_message("Error: %s", e)
+        log_debug_message("Logging the input:")
+        log_debug_message("%s", json.dumps(input_))
```

Comment:
This PR shows a lot of the same changes as the PR of the base branch. Why so? Can you clean this PR to only focus on third party passwordless and passwordless changes?

Reply:
There were merge conflicts and a lot of other complications. I can try. But it will be time consuming.